.travis.yml

```diff
@@ -10,10 +18,13 @@ install:
 - pip install pygments cssselect html5lib lxml
 - git clone --depth=100 --branch=master https://github.com/tabatkins/bikeshed.git ./bikeshed
 - pip install --editable ./bikeshed
-# Use cached copy of bikeshed data files to give consistent builds
-#- bikeshed update
-- cp -R .spec-data/* ./bikeshed/bikeshed/spec-data
-- cp -R .bikeshed-include/* ./bikeshed/bikeshed/boilerplate/webauthn
+- |
+  if "$USE_BIKESHED_CACHE"; then
```

Comment:
nit: style in this file is bash-style `${USE_BIKESHED_CACHE}`, otherwise looks good!

Reply:
Ok, thanks!